commands/generate/component.js

```diff
@@ -24,15 +24,24 @@ module.exports = {
       {
         template: 'component.story.tsx.ejs',
         target: `app/components/${name}/${name}.story.tsx`
-      },
-      {
-        template: 'rollup-index.ts.ejs',
-        target: `app/components/${name}/index.ts`
       }
     ]
 
     await ignite.copyBatch(toolbox, jobs, props)
 
+    // patch the barrel export file
+    const barrelExportPath = `${process.cwd()}/app/components/index.ts`
+    const exportToAdd = `export * from "./${name}/${name}"\n`
+
+    if (!filesystem.exists(barrelExportPath)) {
+      const msg =
+      `No '${barrelExportPath}' file found. Can't export component.` +
+      `Export your new component manually.`
+      print.error(msg)
+      process.exit(1)
```

Comment:
I think this should be a warning and not an error, and allow the script to finish. Because they may be totally okay with exporting their new component manually.